be/src/pipeline/pipeline_x/operator.h

```diff
@@ -465,6 +466,13 @@ class PipelineXSpillLocalState : public PipelineXLocalState<SharedStateArg> {
         return Status::OK();
     }
 
+    void close_spill() {
+        // release memory fast
+        _spill_task.reset();
+    }
+
+    std::shared_ptr<vectorized::SpillTask> _spill_task;
```

Comment:
Do not add this to operator. Because not all operator need spill task.
Operator should be very very abstract.